changelog/Microsoft.DeviceLocalCredentials.json

```
@@ -0,0 +1,38 @@
+{
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "216f8ec6-c6a2-49fc-abff-8d88c19ba8d4",
+          "ApiChange": "Resource",
+          "ChangedApiName": "deviceLocalCredential",
+          "ChangeType": "Addition",
+          "Description": "Added the [deviceLocalCredential](https://learn.microsoft.com/en-us/graph/api/resources/deviceLocalCredential?view=graph-rest-beta) resource type.",
+          "Target": "deviceLocalCredential"
+        },
+        {
+          "Id": "216f8ec6-c6a2-49fc-abff-8d88c19ba8d4",
+          "ApiChange": "Resource",
+          "ChangedApiName": "deviceLocalCredentialInfo",
+          "ChangeType": "Addition",
+          "Description": "Added the [deviceLocalCredentialInfo](https://learn.microsoft.com/en-us/graph/api/resources/deviceLocalCredentialInfo?view=graph-rest-beta) resource",
+          "Target": "deviceLocalCredentialInfo"
+        },
+        {
+          "Id": "216f8ec6-c6a2-49fc-abff-8d88c19ba8d4",
+          "ApiChange": "EntitySet",
+          "ChangedApiName": "deviceLocalCredentials",
+          "ChangeType": "Addition",
+          "Description": "Added the [deviceLocalCredentials](https://learn.microsoft.com/en-us/?view=graph-rest-beta) resource",
+          "Target": "container"
+        }
```

EntitySet isn't useful because we don't document it. I'll file a bug with the onboarding team.
You can replace this object with the two new permissions.
```suggestion
        {
          "Id": "216f8ec6-c6a2-49fc-abff-8d88c19ba8d4",
          "ApiChange": "Permission",
          "ChangedApiName": "DeviceLocalCredential.ReadBasic.All",
          "ChangeType": "Addition",
          "Description": "Added the DeviceLocalCredential.ReadBasic.All delegated and application permissions.",
          "Target": "deviceLocalCredentialInfo"
        },
        {
          "Id": "216f8ec6-c6a2-49fc-abff-8d88c19ba8d4",
          "ApiChange": "Permission",
          "ChangedApiName": "DeviceLocalCredential.Read.All",
          "ChangeType": "Addition",
          "Description": "Added the DeviceLocalCredential.Read.All delegated and application permissions.",
          "Target": "deviceLocalCredentialInfo"
        }
```